docs/source/en/model_doc/siglip.md

```diff
@@ -81,7 +83,8 @@ If you want to do the pre- and postprocessing yourself, here's how to do that:
 >>> url = "http://images.cocodataset.org/val2017/000000039769.jpg"
 >>> image = Image.open(requests.get(url, stream=True).raw)
 
->>> texts = ["a photo of 2 cats", "a photo of 2 dogs"]
+>>> candidate_labels = ["a photo of 2 cats", "a photo of 2 dogs"]
+>>> candidate_labels = [f'This is a photo of {label}.' for label in candidate_labels]  # follows the training data template else the output will be off
```

Comment:
It's not that the output will be off, it's that these will reproduce the outputs from the pipeline

Reply:
Will edit the comment